source/Hardpoint.cpp

```diff
@@ -325,6 +325,7 @@ void Hardpoint::Jam()
 	// Reset the reload count.
 	reload += outfit->Reload();
 	burstReload += outfit->BurstReload();
+	--burstCount;
```

Comment:
```suggestion
	reload = outfit->Reload();
	burstReload = outfit->BurstReload();
	burstCount = 0;
```
On jamming I would expect the weapon to start as if it wasn't loaded. So setting reload and burstReload to the maximum values they have (and not beyond that due to addition), and the burstCount to zero (since the weapon needs a full reload, there are no bursts present).

I didn't test this, but I don't see how just reducing the burstCount would be semantically correct. (And I also feel we should never have longer reload times than outfit=>Reload()).

Reply:
Ok, but if the jam-code does exactly the same as the fire code, then the weapon doesn't really get jammed, but it just fires a dud.

I can imagine that that is the behavior that we want, and in that case it's fine for me to just merge this PR as-is.

Reply:
One small difference I noticed is that the fire code sets `isFiring` to true. I'm not sure what effect that has and if it would also be required.

Reply:
it's not needed to change isFiring here